### DISCUSSION TITLE
What plugins are in use on the home page screen shot?

### DISCUSSION THREAD

**Discussion by gsgleason** [DeaDBeeF-Player/deadbeef#2698] [General]
Hi.  This isn't a problem, but I can't find an appropriate place to ask.

The home page has this image: https://deadbeef.sourceforge.io/title_screenshot.png

It says "The screenshot above demonstrates a highly customized DeaDBeeF, running on Linux with several additional plugins"

I think it would be nice to know which exact plugins are in use.  I'm particularly interested in the track time and information that is right above the player controls, as well as the "Add folders" button and colored player controls.

Thanks!

**Comment by Oleksiy-Yakovenko** [comment]
musical spectrum
waveform seekbar
spectrogram of some sort
some custom layout
playlist browser (built in widget)

I think the playback controls is a standard widget you can use in design mode.
The add folders is a standard button widget, that you can also use in design mode.
Playback information is some 3rd party plugin, but I don't know which one.

**Comment by gsgleason** [comment]
Thanks for your help.

It's the playback status widget!  I was having weird issues because I'm using a plain tiling WM and it was showing HTML.  I'm able to customize it with a right click.  

The waveform seekbar can be customized to show the ruler, which is what's shown.  

I'm not sure how the playlist has the in-line headings, though.  Trying to figure that out.

**Comment by Oleksiy-Yakovenko** [comment]
> I'm not sure how the playlist has the in-line headings, though. Trying to figure that out.

You need to right click on the playlist header, and select Group by -> ...
Then you can also add an album art column.
